dateparser/custom_lang_detect/template.py

```diff
@@ -0,0 +1,32 @@
+"""
+
+IMPORT ALL THE REQUIRED LIBRARIES AND SET CONSTANTS
+
+
+DEFINE A FUNCTION WITH NAME "detect_languages" WHICH ACCEPTS ONE PARAM "text".
+
+
+SEE EXAMPLES BELOW : 
```

Comment:
I'd rather not include this in the code, and also I'd rather tackle the question of user extensibility later, once our internal API is stable.